app/Console/Commands/DevSimulate.php

```diff
@@ -0,0 +1,131 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Console\LnmsCommand;
+use App\Models\Device;
+use Illuminate\Support\Str;
+use LibreNMS\Util\Snmpsim;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Process\Process;
+
+class DevSimulate extends LnmsCommand
+{
+    protected $name = 'dev:simulate';
+    protected $developer = true;
+    /**
+     * @var Snmpsim
+     */
+    protected $snmpsim = null;
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Simulate devices using test data';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this->addArgument('file', InputArgument::OPTIONAL);
+        $this->addOption('multiple', 'm', InputOption::VALUE_NONE);
+        $this->addOption('remove', 'r', InputOption::VALUE_NONE);
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $this->snmpsim = new Snmpsim();
+        $snmprec_dir = $this->snmpsim->getDir();
+        $listen = $this->snmpsim->getIp() . ':' . $this->snmpsim->getPort();
```

Comment:
this should be configurable

Reply:
It never has been before.  I feel like it could be detrimental to have it configurable, if you add multiple devices it won't work if you use different IPs and ports.

Reply:
I maually change it sometimes, so i think it should be configurable

Reply:
`./scripts/save-test-data.php --snmpsim` does not allow that to be changed... this is a replacement for that.